src/ValidatorKeysTool.cpp

```diff
@@ -155,10 +216,24 @@ void printHelp (const boost::program_options::options_description& desc)
         << "Commands: \n"
            "     create_keys        Generate validator keys.\n"
            "     create_token       Generate validator token.\n"
-           "     revoke_keys        Revoke validator keys.\n";
+           "     revoke_keys        Revoke validator keys.\n"
+           "     sign <data>        Sign string with validator key.\n";
```

Comment:
Should the summary on line 214 be updated to `<< name << " [options] <command> [<argument> ...]\n"`?